style.css

```diff
@@ -33,9 +46,19 @@ input {
 
 .instructions {
     position: absolute;
-    top: 470px;
+    top: 0px;
+    left: 850px;
+    margin-left: 10px;
+    margin-right: 10px;
 }
 
 .field_item {
     position: absolute;
 }
+
+.info-area {
+    background-color: rgb(252,221,255);
```

Comment:
I don't know why, but for some reason I prefer hex values. Unless you have a good point for using rgb() notation, could you please convert it to hex? Thanks